docs/builtins/General/constantly.md

```diff
@@ -1,49 +1,49 @@
 ## constantly
-Use `constantly` to lazily ignore the specified arguments `IGNORE` and return the specified `VALUE`.
+
+Use `constantly` to lazily ignore the specified `ignore` arguments and return the specified `vale`.
 
 ### Basic syntax
 
 To lazily ignore arguments and return a value, use the following syntax:
 
-`(constantly value ignore1 ignore2 ignore3 ...)`
+```pact
+(constantly value ignore1 ignore2 ...)
+```
 
 ### Arguments
 
-Use the following arguments to specify the `VALUE` to return and the `IGNORE` arguments for the `constantly` Pact function.
+Use the following arguments to specify the `value` to return and the `ignore` arguments for the `constantly` Pact function.
 
 | Argument | Type | Description |
 | --- | --- | --- |
-| `value` | `<a>` | Specifies the value to return. |
-| `ignore1, ignore2, ignore3, ...` | `<b>, <c>, <d>, ...` | (Optional) Specifies the arguments to ignore. You can have multiple ignore arguments. |
+| `value` | any | Specifies the value to return. |
+| `ignore1, ignore2, ...` | any | (Optional) Specifies the arguments to ignore. You can have multiple ignore arguments. |
 
 ### Return values
 
-The `constantly` function returns the specified `VALUE`, ignoring any additional arguments.
+The `constantly` function returns the specified `value`, ignoring any additional arguments.
 
 ### Examples
 
 The following example demonstrates the `constantly` function in the Pact REPL:
 
 ```pact
-pact>(filter (constantly true) [1 2 3])
+pact> (filter (constantly true) [1 2 3])
 [1 2 3]
 ```
 
-In this example, `constantly true` is used as the predicate function for `filter`. It ignores the elements of the list `[1 2 3]` and always returns `true`, so `filter` returns the entire list.
+In this example, `constantly true` is used as the predicate function for the `filter` function. 
+The `constantly` function always returns `true` and ignores the elements of the `[1 2 3]` list argument, so the `filter` function always returns the entire list.
 
-#### Additional Examples:
-
-Here are a few more examples of using `constantly`:
+The following example illustrate specifying multiple arguments and argument using different data types:
 
 ```pact
-pact>(constantly "hello" 1)
+pact> (constantly "hello" {"name": "Kris", "age": 45})
 "hello"
 
-pact>(constantly "world" "ignore" "these" "arguments")
+pact> (constantly "world" 6 "ignore" "these" "arguments" [2.1 3.0])
```

Comment:
arguably the `6` here defeats the purpose of the mnemonic given by the "ignore these arguments" statement.